archive/prerelease/1.1/index.bs

```diff
@@ -675,11 +675,17 @@ A user agent MAY dispatch this event type to indicate that something has occured
 A user agent MAY dispatch this event type to indicate that something has occured which suggests the {{VRDisplay}} should exit presentation. For example, if the {{VRDisplay}} is capable of detecting when the user has taken it off, this event SHOULD fire when they do so with the reason "unmounted".
 
 <dfn event for="Window" id="window-onvrdisplayblur-event">onvrdisplayblur</dfn>
-A user agent MAY dispatch this event type to indicate that presentation to the display by the page is paused by the user agent, OS, or VR hardware. While a {{VRDisplay}} is blurred it does not lose it's presenting status ({{isPresenting}} continues to report true) but {{getFrameData()}} returns false without updating the provided {{VRFrameData}} and {{getPose()}} returns null. This is to prevent tracking while the user interacts with potentially sensitive UI. For example: A user agent SHOULD blur the presenting application when the user is typing a URL into the browser with a virtual keyboard, otherwise the presenting page may be able to guess the URL the user is entering by tracking their head motions.
+A user agent MAY dispatch this event type to indicate that presentation to the display by the page is paused by the user agent, OS, or VR hardware. While a {{VRDisplay}} is blurred it does not lose it's presenting status ({{isPresenting}} continues to report true) but {{getFrameData()}} returns false without updating the provided {{VRFrameData}} and {{getPose()}} returns a {{VRPose}} with null members. This is to prevent tracking while the user interacts with potentially sensitive UI. For example: A user agent SHOULD blur the presenting application when the user is typing a URL into the browser with a virtual keyboard, otherwise the presenting page may be able to guess the URL the user is entering by tracking their head motions.
 
 <dfn event for="Window" id="window-onvrdisplayfocus-event">onvrdisplayfocus</dfn>
 A user agent MAY dispatch this event type to indicate that presentation to the display by the page has resumed after being blurred.
 
+<dfn event for="Window" id="window-onvrdisplaypointerrestricted-event">onvrdisplaypointerrestricted</dfn>
+A user agent MAY dispatch this event type to indicate that pointer input is restricted to consumption via a <a href="http://w3c.github.io/pointerlock/">pointerlocked</a> element. This event MUST only be dispatched while the {{VRDisplay}} is presenting. Pages that wish to consume pointer input for HMD experiences are encouraged to call requestPointerLock in response to this event.
```

Comment:
Just clarifying: My reading is that this indicates that the page is entering a mode where normal mouse events/pointer events are no longer fired at all unless it's against a pointerlocked element, correct?